feature/office/src/main/java/com/iteneum/office/presentation/ui/Office.kt

```diff
@@ -29,43 +31,55 @@ import com.iteneum.office.R
  * Comment
  */
 @Composable
-fun OfficeUI() {
-
+fun OfficeUI(vm: OfficeViewModel = viewModel()) //Injecting the viewModel
```

Comment:
If you are going to define the function onClick in the viewModel, please invoke the function in the UI